### PR TITLE
Fix bug wrong password throw exception

### PR DESCRIPTION
**Pull request by marknguyen85**
Fix bug wrong password throw exception

### CODE REVIEW

File: src/sagas/swapSaga.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ function *swapToken() {
 
     yield call(fetchTransactionReceipt, txHash);
   } catch (error) {
-    yield put(txActions.setConfirmingError(error));
+    yield put(txActions.setConfirmingError(error.message));
```
Comment (artwooood) [inline src/sagas/swapSaga.js]: This should be leaved as it is for now. We still have other services for importing account that return the error as string, not object.

File: src/services/accountServices/KeystoreService.js [hunk 2]
```diff
@@ -8,7 +8,14 @@ export default class KeystoreService {
   }
 
   sendTransaction = (txObject, password) => {
-    const account = this.web3.eth.accounts.decrypt(this.keystore, password);
+    let account = null;
+    try {
+      account = this.web3.eth.accounts.decrypt(this.keystore, password);
+    }
+    catch (error) {
+      throw error;
```
Comment (artwooood) [inline src/services/accountServices/KeystoreService.js]: Can you throw `error` here as string: `throw error.message`

File: src/services/accountServices/KeystoreService.js [hunk 3]
```diff
@@ -8,7 +8,14 @@ export default class KeystoreService {
   }
 
   sendTransaction = (txObject, password) => {
-    const account = this.web3.eth.accounts.decrypt(this.keystore, password);
+    let account = null;
```
Comment (artwooood) [inline src/services/accountServices/KeystoreService.js:11]: I think the whole process of unlocking account worths to be separated into another function. And here, we just need to call `account = this.unlockAccount()`

**Review by artwooood — CHANGES_REQUESTED** [review]

File: src/services/accountServices/KeystoreService.js (hunk 2 above)
Comment (marknguyen85) [reply src/services/accountServices/KeystoreService.js]: ok

File: src/services/accountServices/KeystoreService.js (hunk 3 above)
Comment (marknguyen85) [reply src/services/accountServices/KeystoreService.js:11]: yes i will implement on next times.

**Review by manhlx3006 — APPROVED** [review]
Look good to me

**Review by artwooood — APPROVED** [review]

**Pull request merged**